cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -77,6 +77,12 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
+  cleanupPolicy:
```

Comment:
These settings could also be added to the docs in ceph-cluster-crd.md.

Reply:
Absolutely; I was planning to add a doc patch after Santosh's PR, but if we're (sensibly) merging the PRs this can certainly go in now.